modules/apps/translation/translation-service/src/main/java/com/liferay/translation/internal/exporter/XLIFF12InfoFormTranslationExporter.java

```diff
@@ -79,33 +84,97 @@ public InputStream exportInfoItemFieldValues(
 
 		Element bodyElement = fileElement.addElement("body");
 
-		Collection<InfoFieldValue<Object>> infoFieldValues =
-			infoItemFieldValues.getInfoFieldValues();
+		Map<String, List<InfoFieldValue<Object>>> infoFieldValuesMap =
+			new LinkedHashMap<>();
+
+		for (InfoFieldValue<Object> infoFieldValue :
+				infoItemFieldValues.getInfoFieldValues()) {
 
-		for (InfoFieldValue<Object> infoFieldValue : infoFieldValues) {
 			InfoField infoField = infoFieldValue.getInfoField();
 
-			if (!_translationInfoFieldChecker.isTranslatable(infoField)) {
-				continue;
+			if (_translationInfoFieldChecker.isTranslatable(infoField)) {
+				List<InfoFieldValue<Object>> infoFieldValuesList =
```

Comment:
You are creating a infoFieldValuesList on the for to add the infoFieldValue. Why?

Reply:
There can be multiple infoFieldValues per infoField. I separated the collecting/traversal steps so that the code explicitly states that there are multiple values per field.

Reply:
I see it now, thanks!
I did not realize that this code where updating the map.